scripts/generate-swagger-ui.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -e
+
+[[ -z "$SWAGGER_UI_VERSION" ]] && echo "missing \$SWAGGER_UI_VERSION" && exit 1
+
+SWAGGER_UI_GIT="https://github.com/swagger-api/swagger-ui.git"
+CACHE_DIR="./.cache/swagger-ui/$SWAGGER_UI_VERSION"
+GEN_DIR="./third_party/OpenAPI"
+
+escape_str() {
+  echo "$1" | sed -e 's/[]\/$*.^[]/\\&/g'
+}
+
+# do caching if there's no cache yet
+if [[ ! -d "$CACHE_DIR" ]]; then
+  mkdir -p "$CACHE_DIR"
+  tmp="$(mktemp -d)"
+  git clone --depth 1 --branch "$SWAGGER_UI_VERSION" "$SWAGGER_UI_GIT" "$tmp"
+  cp -r "$tmp/dist/"* "$CACHE_DIR"
+  cp -r "$tmp/LICENSE" "$CACHE_DIR"
+  rm -rf "$tmp"
+fi
+
+# populate swagger.json
+tmp="    urls: ["
+for i in $(find "$GEN_DIR" -name "*.swagger.json"); do
+  escaped_gen_dir="$(escape_str "$GEN_DIR/")"
+  path="$(echo $i | sed -e "s/$escaped_gen_dir//g")"
+  tmp="$tmp{\"url\":\"$path\",\"name\":\"$path\"},"
+done
+# delete last characters from $tmp
+tmp=$(echo "$tmp" | sed 's/.$//')
+tmp="$tmp],"
+
+# recreate swagger-ui, delete all except swagger.json
+find "$GEN_DIR" -type f -not -name "*.swagger.json" -delete
+mkdir -p "$GEN_DIR"
+cp -r "$CACHE_DIR/"* "$GEN_DIR"
```

Comment:
This still isn't copying the right thing. It's copying the `dist` folder instead of the contents of the dist folder. The error I'm seeing is
```
$ make generate/swagger-ui 
SWAGGER_UI_VERSION=v4.15.5 ./scripts/generate-swagger-ui.sh
cat: ./third_party/OpenAPI/swagger-initializer.js: No such file or directory
sed: can't read ./third_party/OpenAPI/swagger-initializer.js: No such file or directory
make: *** [Makefile:8: generate/swagger-ui] Error 2
```

Can you not make the changes I suggested?
```suggestion
cp -r "$CACHE_DIR"/dist/* "$GEN_DIR"
cp -r "$CACHE_DIR"/LICENSE "$GEN_DIR"
```

This fixes it for me.

Reply:
Hi @johanbrandhorst , I changed it on the caching layer side 

```shell
git clone --depth 1 --branch "$SWAGGER_UI_VERSION" "$SWAGGER_UI_GIT" "$tmp"
cp -r "$tmp/dist/"* "$CACHE_DIR"
cp -r "$tmp/LICENSE" "$CACHE_DIR"
```

Have you delete the `.cache/` before trying it again?

Reply:
I see, yep, it works now that the cache is deleted!